docs/index.rst

```diff
@@ -22,6 +22,7 @@ xclim currently provides over 50 indices related to mean, minimum and maximum da
    api
    notebooks/units
    icclim
+   pcc
```

Comment:
I would put that in an example section.

Reply:
ie: `documentation` --> `examples` --> `Icclim`, `pcc`, etc.?

Reply:
I think ICCLIM has widespread recognition, so OK to keep it in the main namespace. PCC has not the same name recognition, so I would include it as an example/demo of "how to create a custom library of indices".

Reply:
This is dealt with.